ov/storage.go

```diff
@@ -52,17 +52,23 @@ type StoragePath struct {
 	IsEnabled         bool     `json:"isEnabled"`                   // isEnabled (required), Identifies whether the storage path is enabled.
 	Status            string   `json:"status,omitempty"`            // status (read only), The overall health status of the storage path.
 	StorageTargetType string   `json:"storageTargetType,omitempty"` // storageTargetType ('Auto', 'TargetPorts')
-	StorageTargets    []string `json:"targets,omitempty"`           // only set when storageTargetType is TargetPorts
+	Targets           []Target `json:"targets,omitempty"`           // only set when storageTargetType is TargetPorts
 	TargetSelector    string   `json:"targetSelector,omitempty"`
 }
 
+type Target struct {
+	IpAddress string `json:"IpAddress,omitempty"`
+	Name      string `json:"name,omitempty"`
+	TopPort   string `json:"topPort,omitempty"`
```

Comment:
Is it top port or tcp port and what is the data type for the port string or integer?

Reply:
TopPort which is a string according to REST Doc. I wasn't able to get a proper value for these fields in response though.